src/test/java/com/snack/news/service/PicksServiceTest.java

```diff
@@ -0,0 +1,43 @@
+package com.snack.news.service;
+
+import com.snack.news.dto.RequestQueryDto;
+import com.snack.news.exception.PicksNotFoundException;
+import com.snack.news.repository.PicksRepository;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+class PicksServiceTest {
+
+	@InjectMocks
+	private PicksService picksService;
+
+	@Mock
+	private PicksRepository picksRepository;
+
+	@ParameterizedTest
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=4962836)